src/cmd/top/joblist_pane.c

```diff
@@ -207,6 +207,11 @@ static void joblist_continuation (flux_future_t *f, void *arg)
     json_decref (joblist->jobs);
     joblist->jobs = json_incref (jobs);
     joblist_pane_draw (joblist);
+    if (joblist->top->test_exit) {
```

Comment:
commit message "change the behavior of the --test-only" -> "--test-exit" or perhaps "change the behavior of the test option".

Reply:
D'oh! Thanks! Yeah it is the `--test-exit` option :facepalm: